docs/pages/api.rst

```diff
@@ -348,15 +348,19 @@ collapse
 
 Collapse a collection of items into a simple flat collection.
 
+.. warning:: Key differences compared to ``flatten`` are that only one level will be collapsed and values at the bottom level will be removed.
```

Comment:
I looked into the tests to figure this out, please let me know if there are any other differences which should be mentioned

Reply:
That's the proper behavior indeed.